frontends/web/src/routes/account/send/send.tsx

```diff
@@ -158,28 +158,15 @@ class Send extends Component<Props, State> {
     });
 
     this.unsubscribeList = [
-      apiWebsocket((payload) => {
-        if ('type' in payload) {
-          const { data, meta, type } = payload;
-          switch (type) {
-          case 'device':
-            switch (data) {
-            case 'signProgress':
-              this.setState({ signProgress: meta, signConfirm: false });
-              break;
-            case 'signConfirm':
-              this.setState({ signConfirm: true });
-              break;
-            }
-            break;
-          }
-        }
-      }),
+      signProgress((meta) =>
```

Comment:
nit: meta is event internal.. Maybe name it progress or similar?

```suggestion
      signProgress((progress) =>
```